jest.config.mjs

```diff
@@ -57,6 +57,9 @@ export default {
     '/packages/jest-haste-map/src/__tests__/haste_impl.js',
     '/packages/jest-haste-map/src/__tests__/dependencyExtractor.js',
     '/packages/jest-haste-map/src/__tests__/test_dotfiles_root/',
+    '/packages/jest-mock/src/__tests__/class-mocks-types.ts',
+    '/packages/jest-mock/src/__tests__/TestClass.ts',
+    '/packages/jest-mock/src/__tests__/SuperTestClass.ts',
```

Comment:
Thanks! Apologies for that clumsy comment, it was unclear. These can be removed now, right?

```suggestion
```